pymc_experimental/model/transforms/autoreparam.py

```diff
@@ -174,15 +176,18 @@ def vip_reparam_node(
 ) -> Tuple[ModelDeterministic, ModelNamed]:
     if not isinstance(node.op, RandomVariable | SymbolicRandomVariable):
         raise TypeError("Op should be RandomVariable type")
```

Comment:
Can you also raise NotImplementedError for op.ndim_supp>0?

Reply:
it will not pass the dispatch anyway, it is not needed

Reply:
> one lambda per shape or size

Per size for sure 100%